gatsby/content/blog/2020/08/2020-08-27-synapse-1.19.1-released.mdx

```diff
@@ -0,0 +1,35 @@
+---
+date: '2020-08-27'
+title: Synapse 1.19.1 released
+categories:
+  - Releases
+author: Brendan Abolivier
+---
+
+Synappse 1.19.1 is a bug fix release to fix two bugs introduced in 1.19.0.
+
+Those two bugs were related to the new rate-limiter on room joins, which was introduced in 1.19.0.
+
+The first bug caused Synapse to ignore exceptions to rate-limiting for application services.
+
+The second one prevented profile updates from being correctly propagated across rooms.
```

Comment:
```suggestion
The second prevented profile updates from being correctly propagated across rooms.
```

Reply:
I find "the second one" easier to read imho